updaters/github.js

```diff
@@ -0,0 +1,173 @@
+                  (
+                   (!library.version) ||
+                   (
+                    (greaterVer) &&
+                    (
+                     (stable.is(tag)) ||
+                     (!stable.is(tag) && !stable.is(library.version))
+                    )
+                   )
+                  )
+                 ) {
+                console.log('Updated package.json to version'.green, tag);
+                var libraryPath = path.normalize(path.join(__dirname, '../../cdnjs', 'ajax', 'libs', library.name, 'package.json'));
+                var libraryJSON = JSON.parse(fs.readFileSync(libraryPath, 'utf8'));
+                libraryJSON.version = tag;
+                fs.writeFileSync(libraryPath, JSON.stringify(libraryJSON, undefined, 2) + '\n');
+              }
+              async.each(allFiles, function(file, callback) {
+                var fileName = file._;
+                var fileTarget = path.normalize(path.join(__dirname, '../../cdnjs', 'ajax', 'libs', library.name, tag, fileName));
+                fs.ensureFile(fileTarget, function(err) {
+                  if (err) {
+                    console.log('Some strange error occured here'.red);
+                    console.dir(err);
+                    callback();
+                  } else {
+                    rp({
+                      uri:'https://raw.githubusercontent.com/'
+                        + githubRepo(library.autoupdate.target) + '/'
+                        + file.tree + '/' + file.basePath + '/' + fileName,
+                      headers: {
+                        'User-Agent': userAgent
+                      }
+                    })
+                    .then(function (fileData) {
+                      fs.writeFile(fileTarget, fileData, function(err) {
+                        if (err) {
+                          console.dir(err);
+                          console.log('Some strange error occured here'.red);
+                          callback();
+                        } else {
+                          fs.chmodSync(fileTarget, '0644');
+                          callback();
+                        }
+                      });
+                    })
+                    .catch(function (err) {
+                      console.log('error in file retrieval from github');
+                      callback();
+                    });
+                  }
+                });
+              }, function() {
+                callback();
+              });
+            });
+          }, function() {
+            console.log(library.name.green, 'updated from GitHub'.green);
+            callback(null, 1);
+          });
+        })
+        .catch(function (err) {
+          console.log(library.name, 'git tag handle failed');
+          console.dir(err);
+          next();
+        })
+      }
+  ]);
+};
+
+var githubRepo = function (url) {
+  return url.slice(url.indexOf('/', 10)+1, url.indexOf('.git'))
```

Comment:
missing spaces for `+`

Reply:
Is there any other style you would like to see updated before I commit?

Reply:
Are there any other stylistic changes required?

Reply:
I should copy this one here: https://github.com/cdnjs/cdnjs/blob/master/.jscsrc
We have an issue with it: https://github.com/cdnjs/autoupdate/issues/12
Thanks for asking!